### PR TITLE
feat: create table in integration table from evadb select query

### PR DESCRIPTION
**Pull request by gaurav274**
1. `CREATE TABLE postgres_db.table AS SELECT ... `
2. Moved `native_storage_engine` to `sqlalchemy`

### CODE REVIEW

File: test/third_party_tests/test_native_executor.py [hunk 1]
```diff
@@ -83,6 +83,27 @@ def _create_evadb_table_using_select_query(self):
...
+        self.assertEqual(res_batch.frames["derived_table.name"][0], "aa")
+        self.assertEqual(res_batch.frames["derived_table.age"][0], 1)
+        self.assertEqual(res_batch.frames["derived_table.name"][1], "bb")
+        self.assertEqual(res_batch.frames["derived_table.age"][1], 2)
+        execute_query_fetch_all(
+            self.evadb,
+            """USE test_data_source {
+                DROP TABLE IF EXISTS derived_table
+            }""",
+        )
```
Comment (xzdandy) [inline test/third_party_tests/test_native_executor.py:105]: Is the clean up code? If so, shall we put it in the cls teardown?

File: evadb/third_party/databases/sqlite/sqlite_handler.py [hunk 2]
```diff
@@ -146,3 +151,45 @@ def execute_native_query(self, query_string: str) -> DBHandlerResponse:
             return DBHandlerResponse(data=self._fetch_results_as_df(cursor))
         except sqlite3.Error as e:
             return DBHandlerResponse(data=None, error=str(e))
+
+    def _sqlite_to_python_types(self, sqlite_type: str):
```
Comment (xzdandy) [inline evadb/third_party/databases/sqlite/sqlite_handler.py:155]: How about other handlers? Are they going to inherit this one?

File: test/third_party_tests/test_native_executor.py (hunk 1 above)
Comment (gaurav274) [reply test/third_party_tests/test_native_executor.py:105]: I agree. The test class is getting complicated and needs to be simplified.

File: evadb/third_party/databases/sqlite/sqlite_handler.py (hunk 2 above)
Comment (gaurav274) [reply evadb/third_party/databases/sqlite/sqlite_handler.py:155]: Nope, each database has its nuances and has to implement its own.

File: evadb/third_party/databases/sqlite/sqlite_handler.py [hunk 3]
```diff
@@ -146,3 +151,45 @@ def execute_native_query(self, query_string: str) -> DBHandlerResponse:
...
+        mapping = {
```
[the remark this answers is not in the document]
Comment (gaurav274) [reply evadb/third_party/databases/sqlite/sqlite_handler.py:156]: 1. Don't want to expose EvaDB types to third-party integrations.
2. We don't support it right now.

File: evadb/third_party/databases/sqlite/sqlite_handler.py [hunk 4]
```diff
@@ -146,3 +151,45 @@ def execute_native_query(self, query_string: str) -> DBHandlerResponse:
...
+            "DOUBLE PRECISION": float,
+            "FLOAT": float,
+            "NUMERIC": float,
+            "DECIMAL": float,
+            "BOOLEAN": bool,
+            "DATE": datetime.date,
+            "DATETIME": datetime.datetime,
+        }
+
+        def preprocess_sqlite_type(sqlite_type):
```
[the remark this answers is not in the document]
Comment (gaurav274) [reply evadb/third_party/databases/sqlite/sqlite_handler.py:186]: I don't need a separate local function anymore. I can directly call the split on the input of the outer function.

**Review by xzdandy — APPROVED** [review]

**Pull request merged**